src/navigation/wallet/components/RecoveryPhrase.tsx

```diff
@@ -557,8 +561,10 @@ const RecoveryPhrase = () => {
   }, []);
 
   return (
-    <ScrollViewContainer>
-      <ImportContainer>
+    <ScrollViewContainer
+      extraScrollHeight={50}
```

Comment:
Since the recovery phrase input is larger than the memo input, more `extraScrollHeight` is needed to ensure the input is fully above the keyboard. Seems like `90` is an optimal value (works well on an iPhone SE)

Reply:
Great! It looks better. 

Thanks!